### PR TITLE
Update Changelog for release 2022.02.0

### PR DESCRIPTION
**Pull request by manics**
With @yuvipanda's R improvements I think it's time for a release!

Created with
`github-activity -s 2021.08.0 | sed -r -e 's%\[#([0-9]+)\]\(https://github.com/jupyterhub/repo2docker/pull/[0-9]+\)%:pr:`\1`%' -e 's%\(\[\@([A-Za-z0-9-]+)\]\(https://github.com/[A-Za-z0-9-]+\)\)%by :user:`\1`%' > output.txt`

followed by some editing of `output.txt`

### CODE REVIEW

**Comment by yuvipanda** [comment]
I think we've merged a few more PRs after this. Let's get those in and we can make a release?

**Comment by manics** [comment]
I'd like to get https://github.com/manics/jupyter-offlinenotebook/pull/389 in (conda package is building now) and maybe some of the PRs opened today.

**Comment by manics** [comment]
Rebased and updated. I've included these pending PRs:
- [ ] #1123
- [ ] #1124
- [ ] #1227

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,54 @@ Changelog
...
+- Allow passing in extra args to Docker initialization :pr:`1124` by :user:`yuvipanda`
+- Allow passing in traitlets via commandline :pr:`1123` by :user:`yuvipanda`
+- Bump default R version to 4.1 :pr:`1107` by :user:`yuvipanda`
+- Update jupyterlab 3.2.5 jupyter-resource-usage 0.6.1 :pr:`1105` by :user:`manics`
+- Get binary R packages from packagemanager.rstudio.com :pr:`1104` by :user:`yuvipanda`
+- Support R 4.1 :pr:`1102` by :user:`yuvipanda`
+- Add command line option to pass extra build args :pr:`1100` by :user:`TimoRoth`
+- Set labels when building image from Dockerfile :pr:`1097` by :user:`TimoRoth`
+- jupyterlab 3.1.17 :pr:`1092` by :user:`minrk`
+- Bump JupyterLab to 3.1.11 :pr:`1081` by :user:`choldgraf`
```
Comment (manics) [inline docs/source/changelog.rst:24]:
```suggestion
- Bump JupyterLab to 3.1.11 :pr:`1081` by :user:`choldgraf`
- Bootstrap base env with micromamba :pr:`1062` by :user:`wolfv ` 
```

**Comment by manics** [comment]
Potential issue here https://github.com/jupyterhub/repo2docker/pull/1062#issuecomment-1023073794

Should we wait, or release?

**Comment by yuvipanda** [comment]
@manics definitely wait I'd say.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,54 @@ Changelog
...
+---------
+
+Other merged PRs
+----------------
+
+- Remove deprecated calls to distutils :pr:1122 by :user:minrk
+- Delete /tmp/downloaded_packages after running install.R :pr:1119 by :user:yuvipanda
+- Use a smaller R library in our tests :pr:1118 by :user:yuvipanda
+- Only get R itself (r-base-core) from apt, not CRAN packages :pr:1117 by :user:minrk
+- set USER root after each directive block :pr:1115 by :user:minrk
```
Comment (manics) [inline docs/source/changelog.rst:40]:
```
- Put micromamba in /usr/local/bin and use mamba for installs :pr:`1128` by :user:`minrk`
- Remove deprecated calls to distutils :pr:`1122` by :user:`minrk`
- Delete /tmp/downloaded_packages after running install.R :pr:`1119` by :user:`yuvipanda`
- Use a smaller R library in our tests :pr:`1118` by :user:`yuvipanda`
- Only get R itself (r-base-core) from apt, not CRAN packages :pr:`1117` by :user:`minrk`
- set USER root after each directive block :pr:`1115` by :user:`minrk`
```

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -3,6 +3,54 @@ Changelog
 =========
 
 
+Version 2022.01.0
```
Comment (manics) [inline docs/source/changelog.rst:6]:
```suggestion
Version 2022.02.0
```

**Review by minrk — APPROVED** [review]
With #1128 having fixed the rough edges around the micromamba update, I think this is ready to go, unless there's anything else folks want to finish up.

**Comment by manics** [comment]
If no-one says anything I'll make a release end of this week/start of next.

**Comment by minrk** [comment]
👍 from me. I won't do any more merging until then unless it's specifically brought up here.

**Comment by manics** [comment]
Released!

**Comment by yuvipanda** [comment]
@manics \o/ amazing! THANK YOU!

**Pull request merged**